### PR TITLE
Removes all Quirks from Command and Sec

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting your Pull Request
Removes all Quirks from Command and Sec
# Wiki Documentation
🆑
Tweak: Removes all Quirks from Command and Sec
/🆑

### CODE REVIEW

**Comment by patpol4** [comment]
hugbox

File: code/datums/traits/_quirk.dm [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 /datum/quirk/New(mob/living/quirk_mob, spawn_effects, no_init = FALSE)
 	..()
+	if(IS_COMMAND(quirk_mob) | IS_SECURITY(quirk_mob))
+		return
```
Comment (Bibby0110) [inline code/datums/traits/_quirk.dm:20]: You’ll have to qdel(src) before you return

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Comment by JamieD1** [comment]
> hugbox

So you want a no legged security officer with bad aim who is deaf, addicted to meth and is blind trying to arrest antags.

**Comment by Bibby0110** [comment]
> > hugbox
> 
> So you want a no legged security officer with bad aim who is deaf, addicted to meth and is blind trying to arrest antags.

Ye

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Also template

**Comment by JamieD1** [comment]
> hugbox

Opposite of hugbox.. this stops snowflakes..

File: code/datums/traits/_quirk.dm (hunk 1 above)
Comment (JamieD1) [inline code/datums/traits/_quirk.dm]:
```suggestion
	if(IS_COMMAND(quirk_mob) | IS_SECURITY(quirk_mob))
	    qdel(src)
		return
```

File: code/datums/traits/_quirk.dm [hunk 2]
```diff
@@ -15,6 +15,9 @@
 
 /datum/quirk/New(mob/living/quirk_mob, spawn_effects, no_init = FALSE)
 	..()
+	if(IS_COMMAND(quirk_mob) | IS_SECURITY(quirk_mob))
```
Comment (adamsong) [inline code/datums/traits/_quirk.dm]:
```suggestion
	if(IS_COMMAND(quirk_mob) || IS_SECURITY(quirk_mob))
```

**Review by adamsong — APPROVED** [review]

**Review by adamsong — CHANGES_REQUESTED** [review]
Last comment was supposed to request changes, please convert spaces to tabs thanks

**Comment by yacabo** [comment]
I don't think this is a problem nearly enough to demand removing quirks outright

**Comment by adamsong** [comment]
We could always just ban sec/command for incompetence if their quirks interfere with their role

**Comment by maxion12345** [comment]
jamie wants to remove officer wheels for being overpowered

**Comment by JamieD1** [comment]
11 downvotes. Oof

**Comment by adamsong** [comment]
12 downvotes, no upvotes, no reason to fully remove all quirks

**Pull request closed without merging**